client/www/pages/status.tsx

```diff
@@ -0,0 +1,399 @@
+      monitors,
+      overall_uptime: overallUptime,
+      last_updated: new Date().toISOString(),
+    };
+  };
+
+  const fetchUptimeData = async () => {
+    try {
+      const response = await fetch('/api/uptime');
+      const rawData = await response.json();
+      const processedData = processUptimeData(rawData);
+      setUptimeData(processedData);
+      setLastUpdated(new Date());
+      setNextUpdate(60);
+      setLoading(false);
+    } catch (error) {
+      console.error('Failed to fetch uptime data:', error);
+      setLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      setNextUpdate((prev) => {
+        if (prev <= 1) {
+          fetchUptimeData();
+          return 60;
+        }
+        return prev - 1;
+      });
+    }, 1000);
+
+    return () => clearInterval(timer);
+  }, []);
+
+  useEffect(() => {
+    fetchUptimeData();
+  }, []);
+
+  useEffect(() => {
+    const animate = () => {
+      setSmall(false);
+      setTimeout(() => {
+        setSmall(true);
+        setTimeout(() => {
+          animate();
+        }, 3000);
+      }, 100);
+    };
+    animate();
+  }, []);
+
+  const backendMonitor = uptimeData?.monitors?.find(
+    (m: any) =>
+      m.friendly_name?.toLowerCase().includes('backend') ||
+      m.friendly_name?.toLowerCase().includes('api'),
+  );
+  const walMonitor = uptimeData?.monitors?.find(
+    (m: any) =>
+      m.friendly_name?.toLowerCase().includes('wal') ||
+      m.friendly_name?.toLowerCase().includes('write'),
+  );
+
+  const allOperational =
+    uptimeData?.monitors?.length > 0 &&
+    uptimeData?.monitors?.every((m: any) => m.status === 2);
+  return (
+    <div className="flex flex-col relative min-h-screen overflow-y-auto">
+      <div className="flex justify-center px-4 sm:px-8 md:px-16 lg:px-32 xl:px-64 z-10 py-4 md:py-8 relative">
+        <div className="relative bg-white w-full max-w-4xl h-32 sm:h-44 md:h-60 border-2 border-gray-200">
+          <div className="absolute top-3 right-4 text-xs md:text-sm font-mono">
+            <div className="text-right">
+              <div className="font-semibold text-gray-700 mb-1">
+                Service Status
+              </div>
+              <text className="text-gray-500">
+                Last updated {lastUpdated.toLocaleTimeString()} | Next update in{' '}
+                {nextUpdate} sec.
+              </text>
+            </div>
+          </div>
+          <div className="flex gap-4 sm:gap-6 md:gap-8 h-full justify-center items-center px-4">
+            <view
+              className={`flex justify-center items-center ${uptimeData && !loading && allOperational ? 'bg-green-400' : 'bg-gray-400 w-5 h-5 sm:h-7 sm:w-7'} h-10 w-10 sm:h-12 sm:w-12 rounded-full shadow-2xl flex-shrink-0`}
+            >
+              <view
+                className={`${uptimeData && !loading && allOperational ? 'bg-green-400' : 'bg-none'} h-8 w-8 sm:h-10 sm:w-10 absolute rounded-full shadow-2xl`}
```

Comment:
> uptimeData && !loading && allOperational

Wouldn't `allOperational` guarantee `uptimeData && !loading`? 

Maybe easier to write `allOperational` ?